declarations/YouTube.json

```
@@ -9,6 +9,66 @@
     "Terms of Service": {
       "fetch": "https://www.youtube.com/t/terms",
       "select": "div#main-content"
+    },
+    "Community Guidelines": {
+      "fetch": "https://www.youtube.com/intl/ALL_fr/howyoutubeworks/policies/community-guidelines/",
+      "select": [".ytr-hero__details", ".ytr-article__content-wrapper"],
+      "remove": [".ytr-hero__eyebrow", ".ytr-link > svg"]
+    },
+    "Community Guidelines - Self-harm": {
+      "fetch": "https://support.google.com/youtube/answer/2802245",
+      "select": [".article-container"],
+      "remove": [".scClientFeedbackInlineFeedbacklinkroot"]
+    },
+    "Community Guidelines - Hate Speech": {
+      "fetch": "https://support.google.com/youtube/answer/2801939",
+      "select": [".article-container"],
+      "remove": [".scClientFeedbackInlineFeedbacklinkroot"]
+    },
+    "Community Guidelines - Child Sexual Exploitation": {
+      "fetch": "https://support.google.com/youtube/answer/2802002",
```

Cette page me semble plutôt être _Adult Nudity_. Il y a certes un paragraphe sur les mineurs, mais ça ne me semble pas être le sujet principal.

J'aurais plutôt tendance à suggérer [Règlement concernant la sécurité des enfants](https://support.google.com/youtube/answer/2801999?hl=fr&ref_topic=9282679), qui vise plus largement l'exploitation des mineur‧e‧s, pas seulement sous l'angle sexuel.

```suggestion
      "fetch": "https://support.google.com/youtube/answer/2801999",
```

Je me demande si on ne ferait d'ailleurs pas mieux d'élargir et de mettre _Child Exploitation_ comme doctype 🤔 Je vais regarder les autres exemples qu'on a.

---

URL corrigée, merci !

Tout à fait d'accord pour changer `Child Sexual Exploitation` en `Child Exploitation`.